### ISSUE TITLE
Black normals

### ISSUE THREAD

**Issue by abgenullt** [godotengine/godot#62749]
### Godot version

4 alpha 11

### System information

Windows 10, Vulkan,  RX560

### Issue description

Hi,

I've recently got a problem with normal maps.
I tried different model formats (gltf, fbx) and normal maps but can't get rid of this black sides of the model.

Is this a bug or is there a workaround?

Regards
abgenullt

### Steps to reproduce

Import a model and a normal map. Assign a new material with a normal map to it.

### Minimal reproduction project

![Screenshot](https://user-images.githubusercontent.com/34768615/177403500-c9a07005-f0cd-41ed-b190-2fccba4148e4.PNG)

[test.zip](https://github.com/godotengine/godot/files/9049009/test.zip)

**Comment by abgenullt** [comment]
I still have this problem with version 4.1.1. Can someone confirm this issue?

**Comment by Calinou** [comment]
I've never been able to reproduce this in any of my projects, nor the official demo projects. The issue is likely with the source model itself.

**Comment by Stwend** [comment]
When I open the test project with Godot 4.1 stable, the normal map is detected and displayed as what seems to be a red-and-alpha channel icon in the file browser and as a regular RG-compressed normal map in the inspector.
The scene itself looks fine when enabling the default environment and sun - the only difference between the models seems to be a slightly darker shadow color:
![Godot_Normalmap_Artifacts](https://github.com/godotengine/godot/assets/11140839/1a7366ce-4ea4-4733-8405-afb28d47db44)

Same goes for 4.1.2 RC1. My system specs are: Win11, RTX 3070 Ti.

**Comment by abgenullt** [comment]
Thank you for your affort.. I really don't know why but after I deleted and re-downloaded the project, the problem was gone...

**Comment by abgenullt** [comment]
I will test more models and check, if I can find the reason..

**Comment by Calinou** [comment]
This may be due to the same cause as https://github.com/godotengine/godot/issues/64854.

**Comment by abgenullt** [comment]
I don't think so. I have this problem on freshly imported models, too. If I find a the reason, I will let you know.

**Comment by abgenullt** [comment]
Ok, that is very strange. If I copy the working model from the test project to a new, blank project and activate the normal map. I will get the same black normals on one side again..
![image](https://github.com/godotengine/godot/assets/34768615/6b1b33ae-ff56-4d91-960b-a4065479a093)

I get the same effect on two complete different systems. 
I really don't know what is going on here..
[test2.zip](https://github.com/godotengine/godot/files/12813817/test2.zip)
